### ISSUE TITLE
Question..

### ISSUE THREAD

**Issue by chartgod** [wgcban/ChangeFormer#103]
When evaluating performance such as TP, TF, etc., do you evaluate performance by pixel difference between predict_label and correct label?

Is there a source code that evaluates performance among the source codes?

**Comment by chartgod** [comment]
initialize network with normal
cuda:0
================ (Tue Apr 23 17:40:35 2024) ================
gpu_ids: [0] project_name: ChangeFormer checkpoint_root: checkpoints vis_root: vis num_workers: 2 dataset: CDDataset data_name: LEVIR batch_size: 16 split: train split_val: val img_size: 256 shuffle_AB: False n_class: 2 embed_dim: 64 pretrain: None multi_scale_train: False multi_scale_infer: False multi_pred_weights: [0.5, 0.5, 0.5, 0.8, 1.0] net_G: base_transformer_pos_s4_dd8 loss: ce optimizer: sgd lr: 0.01 max_epochs: 1000 lr_policy: linear lr_decay_iters: 100 checkpoint_dir: checkpoints/ChangeFormer vis_dir: vis/ChangeFormer

training from scratch...


lr: 0.0100000

  0%|                                                   | 0/445 [00:00<?, ?it/s]Segmentation fault (core dumped)

Why do these errors occur?
